index.js

```diff
@@ -67,10 +64,10 @@ S3Storage.prototype._handleFile = function (req, file, cb) {
         if (ev.total) currentSize = ev.total
       })
 
-      upload.send(function (err) {
+      upload.send(function (err, data) {
         if (err) return cb(err)
 
-        cb(null, { size: currentSize, key: filePath })
+        cb(null, extend(data, { size: currentSize, key: key }))
```

Comment:
I'm quite much against this change, what is that you want from `data`? Doesn't it contain very much information?